deploy/kubernetes/external-health-monitor-controller/deployment.yaml

```diff
@@ -0,0 +1,51 @@
+# This YAML file demonstrates how to deploy the external
+# health monitor controller for use with the mock CSI driver. It
+# depends on the RBAC definitions from rbac.yaml.
+
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: csi-external-health-monitor-controller
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      external-health-monitor-controller: mock-driver
+  template:
+    metadata:
+      labels:
+        external-health-monitor-controller: mock-driver
+    spec:
+      serviceAccount: csi-external-health-monitor-controller
+      containers:
+        - name: csi-external-health-monitor-controller
+          image: quay.io/k8scsi/csi-external-health-monitor-controller:canary
+          args:
+            - "--v=5"
+            - "--csi-address=$(ADDRESS)"
+            - "--leader-election"
+          env:
+            - name: MY_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
+            - name: ADDRESS
+              value: /var/lib/csi/sockets/pluginproxy/mock.socket
+          imagePullPolicy: "IfNotPresent"
+          volumeMounts:
+            - name: socket-dir
+              mountPath: /var/lib/csi/sockets/pluginproxy/
+
+        - name: mock-driver
+          image: quay.io/k8scsi/mock-driver:canary
```

Comment:
Are you going to submit a PR in csi-test to implement volume health in the mock-driver?  Otherwise we can't test this?

Reply:
I will test this on my local env first, and can send a PR in csi-test if needed